backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -438,11 +439,11 @@ export class AdminResolver {
     } = {
       userId,
```

Comment:
```suggestion
      userId,
      redeemedBy: null,
      validUntil: MoreThan(new Date()),
```
What about setting the defaults (no filters) here?

Reply:
Yes, that's better ⭐️